Adds missing link option necessary for openmp linking.
The target_link_libraries used for amgx are not sufficient as CMake haven't
sorted this out yet, thus manually adding the flag for GCC.
author: micketeer@gmail.com
--- CMakeLists.txt.orig	2024-03-30 03:27:22.354168764 +0100
+++ CMakeLists.txt	2024-03-30 03:28:00.397486779 +0100
@@ -291,6 +291,8 @@
 ELSE (WIN32)
   target_link_libraries(amgx CUDA::cublas CUDA::cusparse CUDA::cusolver CUDA::nvToolsExt m pthread)
   target_link_libraries(amgxsh CUDA::cublas CUDA::cusparse CUDA::cusolver CUDA::nvToolsExt m pthread)
+  target_link_options(amgx INTERFACE "-fopenmp")
+  target_link_options(amgxsh INTERFACE "-fopenmp")
 ENDIF(WIN32)
 
 if(MPI_FOUND)
